fastlane/Fastfile

```diff
@@ -312,6 +311,44 @@ platform :ios do
     end
   end
 
+  desc 'Upload iOS Mattermost.ipa file to s3'
+  lane :upload_ipa_to_s3 do
```

Comment:
maybe make this a generic function outside of the :platform and pass the filename as a parameter to avoid duplicate code

Reply:
Done. Updated `upload_file_to_s3` making it generic when uploading file to S3